src/Sentry.Maui.CommunityToolkitMvvm/CtMvvmMauiElementEventBinder.cs

```diff
@@ -0,0 +1,151 @@
+                TryBindTo(button.Command, bind);
+                break;
+
+            case ImageButton imageButton:
+                TryBindTo(imageButton.Command, bind);
+                break;
+
+            case CarouselView carousel:
+                TryBindTo(carousel.CurrentItemChangedCommand, bind);
+                TryBindTo(carousel.RemainingItemsThresholdReachedCommand, bind);
+                TryBindTo(carousel.PositionChangedCommand, bind);
+                break;
+
+            case CollectionView collectionView:
+                TryBindTo(collectionView.RemainingItemsThresholdReachedCommand, bind);
+                TryBindTo(collectionView.SelectionChangedCommand, bind);
+                break;
+
+            case Entry entry:
+                TryBindTo(entry.ReturnCommand, bind);
+                break;
+
+            case RefreshView refresh:
+                TryBindTo(refresh.Command, bind);
+                break;
+
+            case SearchBar searchBar:
+                TryBindTo(searchBar.SearchCommand, bind);
+                break;
+
+            default:
+                TryGestureBinding(element, bind);
+                break;
+        }
+    }
+
+    private static void TryGestureBinding(VisualElement element, bool bind)
+    {
+        if (element is IGestureRecognizers gestureRecognizers)
+        {
+            foreach (var gestureRecognizer in gestureRecognizers.GestureRecognizers)
+            {
+                TryBindTo(gestureRecognizer, bind);
+            }
+        }
+    }
+
+    private static void TryBindTo(IGestureRecognizer recognizer, bool bind)
+    {
+        switch (recognizer)
+        {
+            case TapGestureRecognizer tap:
+                TryBindTo(tap.Command, bind);
+                break;
+
+            case SwipeGestureRecognizer swipe:
+                TryBindTo(swipe.Command, bind);
+                break;
+
+            // no commands
+            //case PinchGestureRecognizer pinch
+            //case PanGestureRecognizer pan
+            case DragGestureRecognizer drag:
+                TryBindTo(drag.DragStartingCommand, bind); // unlikely to ever be async
+                TryBindTo(drag.DropCompletedCommand, bind);
+                break;
+
+            case PointerGestureRecognizer pointer:
+                TryBindTo(pointer.PointerPressedCommand, bind);
+                TryBindTo(pointer.PointerReleasedCommand, bind);
+                TryBindTo(pointer.PointerEnteredCommand, bind); // unlikely to ever be async
+                TryBindTo(pointer.PointerExitedCommand, bind);
+                break;
+        }
+    }
+
+    private static List<IAsyncRelayCommand> _refs = [];
+    private static void TryBindTo(ICommand? command, bool bind)
+    {
+        if (command is IAsyncRelayCommand relayCommand)
+        {
+            if (bind)
+            {
+                // necessary for collectionview buttons
+                if (!_refs.Contains(relayCommand))
```

Comment:
Is this just used to look up if it was already added? If so shouldn't we use a map here instead? 
right now this is O(n)

Another question: Is the same reference being pass in twice? Or can we rely on `IAsyncRelayCommand` having some `Equals` and `GetHashCode` overridden?

Reply:
When the CollectionView is rebinding cells, this fires off events that we pick up - and WE think we're rebinding.  The issue is that collectionview is going to give you recycled cells (as it should).  We end up X binding the object.  I had a thought over the weekend of NOT storing a reference at all.  My thinking is to simply force an event unhook (-=) even if we haven't hooked it before installing a new hook.  This will prevent double binding as well as us having to look up references.  

Second question: it would be the same issue I mentioned above.  It technically is the same cell reference because of cell recycling.  In any case, with the above potential solution.... none of this matters.  

Thoughts?